### ISSUE TITLE
Extending OIN to include multiple license types

### ISSUE THREAD

**Issue by smit1678** [openimagerynetwork/oin-metadata-spec#15]
To continue the discussion from https://github.com/hotosm/OpenAerialMap/issues/63, there is a proposal on the table to expand the license types within OIN to include CC-BY-NC. 

A number of providers contribute open data through a number of licenses -- CC0, CC-BY, CC-BY-NC, CC-BY-SA. By limiting the license type within OIN, we limit the amount of imagery available. We can of course technically capture multiple licenses and still allow any provider to publish via a very open license or one that restricts usage. 

Main question that seems like should be answered first: 

  * What licenses are most appropriate for open EO data? Can OSM permission be integrated?
  * What would be the standard reference set of open licenses that can be used that OIN could promote through it's metadata standard? 

cc @mojodna @wonderchook @cgiovando @cholmes @smathermather @ianschuler @lossyrob @danlopez00

**Comment by sethfitz** [comment]
FWIW, [SPDX](https://spdx.org/licenses/) has a list of machine-readable identifiers for licenses (this is what NPM uses). We'd probably want to use those in the metadata to effectively normalize the list.

**Comment by smit1678** [comment]
To follow up here. We recently had an OIN technical discussion and had this ticket on the agenda for discussion. In short, it was agreed that the Open Imagery Network should stay as a single license and not open up to a range of licenses. This will make it easier to use and interact with. This also enables other growth outside of the OIN around standards and tool development. 

To recap the discussion about this ticket: 

  - We discussed the pros and cons of extending but everyone was in agreement that OIN should be locked down on a single license. Having a range of licenses is too confusing and diverges from the core idea of making it dead simple and easy to get access to imagery. 
  - We agreed upon the idea of investigating further of what that license is. CC-BY right now may not be best option. Radiant referenced some work that is happening to look at an open image license. 
  - Implementations like OAM can continue to dump imagery into an OIN bucket that is a single license, or it can also pull in imagery that is licensed differently. This isn’t in conflict with the OIN idea. OAM will pick up this conversation back in https://github.com/hotosm/OpenAerialMap/issues/63. 

Notes and agenda from the discussion here: https://docs.google.com/document/d/1RrznXFrRlj9t_S5iC7Atjztg3WAWvqGXmYWx5lCLDAo/edit?usp=sharing. 

Closing this thread out. Add other discussion notes as needed here.
